### DISCUSSION TITLE
Interoperability with fidget-nvim

### DISCUSSION THREAD

**Discussion by mightyiam** [neovide/neovide#1502] [Q&A]
I just installed this LSP event notifier plugin, https://github.com/j-hui/fidget.nvim .

In a terminal I didn't experience the issue. In neovide the notifications animate in a way that is less than pleasant.

[Here's a video](https://photos.app.goo.gl/BBiDLztHXm4YcaFe9).

Is there any way to prevent this, please?

**Comment by ttytm** [comment]
Hey @mightyiam,

You haven't shared a Screenshot or something. That's why I can only guess.

But you are probably experiencing this using neovide in multrigrid mode. You can try using it without it and see if it solves it. 

If this is the issue, it's just that wheen using neovims multrigrid, boundaries are drawn differently. The only way around it would be to use neovide without it or when the developer cares implementing a fix for this specific case. Implement it yourself and submitting a PR did it for me for other plug-ins 👍

**Reply by mightyiam** [reply]
Thank you, @tobealive . I added a video to the description. Does this make the cause of the issue clear?
